<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(145, 145, 145);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 230, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="serialPortComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 153, 230);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recordButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 153, 230);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 153, 230);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 153, 230);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Save </string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomInButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 153, 230);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Zoom In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomOutButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 153, 230);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Zoom Out</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="waveformGraphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>51</y>
      <width>1151</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="QStatusBar1">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
